### DISCUSSION TITLE
Can't edit the article after hosting it on Vercel, even though it works locally

### DISCUSSION THREAD

**Discussion by silvianaim02** [Thinkmill/keystatic#775] [Q&A]
Can't edit the article after hosting it on Vercel (error message : _frontmatter not found_) , even though it works locally. Below I have attached a screenshot : 

![Screenshot 2023-11-22 092248](https://github.com/Thinkmill/keystatic/assets/55119572/442757fa-dcbf-40e4-98c9-3112f1cbe113)

**Comment by simonswiss** [comment]
Hey!

Sorry about the delay in responding. Are you able to share access to the codebase?

**Reply by silvianaim02** [reply]
Thank you for replying @simonswiss  , here is the codebase :

[https://github.com/silvianaim02/silvianaimcom](https://github.com/silvianaim02/silvianaimcom)
